core/domain/exp_domain_test.py

```diff
@@ -2403,6 +2404,41 @@ def test_can_undergo_classification(self):
         self.assertFalse(
             state_without_training_data.can_undergo_classification())
 
+    def test_get_training_data(self):
+        """Test retrieval of training data."""
+        exploration_id = 'eid'
+        test_exp_filepath = os.path.join(
+            feconf.TESTS_DATA_DIR, 'string_classifier_test.yaml')
+        yaml_content = utils.get_file_contents(test_exp_filepath)
+        assets_list = []
+        exp_services.save_new_exploration_from_yaml_and_assets(
+            feconf.SYSTEM_COMMITTER_ID, yaml_content, exploration_id,
+            assets_list)
+
+        exploration = exp_services.get_exploration_by_id(exploration_id)
+        state = exploration.states['Home']
+
+        expected_training_data = []
```

Comment:
@seanlip . The training data is really huge for that particular exploration. Not sure how we can hardcode it here.